BibBuddy/app/src/androidTest/java/de/bibbuddy/NotesFragmentTest.java

```diff
@@ -67,45 +96,62 @@ public void notesFragmentIsDisplayed_Test() {
   }
 
   @Test
-  public void noteListIsDisplayed_Test() {
-    ViewActions.closeSoftKeyboard();
-    onView(withId(idText)).check(matches(isDisplayed()));
-    onView(withId(R.id.noteModDate)).check(matches(isDisplayed()));
-    onView(withId(R.id.noteName)).check(matches(isDisplayed()));
-    onView(withId(R.id.noteType)).check(matches(isDisplayed()));
-    new DrawableMatcher(R.drawable.document)
-        .matchesSafely(itemView);
+  public void recyclerViewWorks_Test() {
+    if (itemView != null) {
+      onView(withId(R.id.note_list_recycler_view)).check(matches(isDisplayed()));
+    }
   }
 
   @Test
-  public void editorIsOpenedOnItemClick_Test() {
-    ViewActions.closeSoftKeyboard();
-    onView(withId(idText)).perform(click());
-    onView(withId(R.id.fragment_text_note_editor)).check(matches(isDisplayed()));
+  public void noteItemIsDisplayedCorrectly_Test() {
+    if (itemView != null) {
+      onView(withId(itemViewId)).check(matches(isDisplayed()));
+
+      int modDateId = itemView.findViewById(R.id.note_mod_date).getId();
+      onView(withId(modDateId)).check(matches(isDisplayed()));
+
+      if (nF.noteList.get(nF.noteList.size() - 1).getType() == NoteTypeLut.TEXT) {
+        int nameId = itemView.findViewById(R.id.note_name).getId();
+        onView(withId(nameId)).check(matches(isDisplayed()));
+
+        new DrawableMatcher(R.drawable.document)
+            .matchesSafely(itemView);
+      } else {
+        new DrawableMatcher(R.drawable.microphone)
+            .matchesSafely(itemView);
+
+        onView(withId(R.id.seekBar)).check(matches(isDisplayed()));
+        onView(withId(R.id.play_note)).check(matches(isDisplayed()));
+        onView(withId(R.id.stop_note)).check(matches(isDisplayed()));
+        onView(withId(R.id.played_time)).check(matches(isDisplayed()));
+        onView(withId(R.id.total_time)).check(matches(isDisplayed()));
+      }
+    }
   }
 
   @Test
-  public void itemIsDeletedOnItemSwipeLeft_Test() {
-    onView(withId(idText)).perform(swipeLeft());
-    new DrawableMatcher(R.color.alert_red).matchesSafely(itemView);
-    new DrawableMatcher(R.drawable.delete).matchesSafely(itemView);
+  public void emptyNoteListViewIsDisplayedCorrectly_Test() {
+    if (itemView == null) {
+      onView(withId(R.id.empty_notes_list_view)).check(matches(isDisplayed()));
+    }
   }
 
   @Test
-  public void deleteSnackbarIsDisplayedOnSwipeLeft_Test() {
-    onView(withId(idText)).perform(swipeLeft());
-    onView(withText(R.string.delete_notification)).check(matches(isDisplayed()));
-    onView(withText(R.string.undo)).check(matches(isDisplayed()));
+  public void textNoteEditorIsOpenedOnItemClick_Test() {
+    if (itemView != null) {
+      if (nF.noteList.get(nF.noteList.size() - 1).getType() == NoteTypeLut.TEXT) {
+        onView(withId(itemViewId)).perform(click());
```

Comment:
I suggest adding a new line before and after the if()-block.

Reply:
Done.